notebooks/charactercodex_llamafile.ipynb

```diff
@@ -0,0 +1,2607 @@
+{
```

Comment:
<p>I'm thinking we should start getting used to having some action items at the end here, maybe link to the tutorials and the repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I added the most relevant resources. Feel free to suggest something better...</p>